welcome/index.adoc

```diff
@@ -29,7 +29,7 @@ ifndef::openshift-rosa,openshift-dedicated,openshift-dpu,openshift-telco[]
 To navigate the {product-title} {product-version} documentation, you can use one of the following methods:
 
 * Use the navigation bar to browse the documentation.
-* Select the task that interests you from the contents of the xref:../welcome/learn_more_about_openshift.adoc#learn_more_about_openshift[Learn more about {product-title}]
+* Select the task that interests you from xref:../welcome/learn_more_about_openshift.adoc#learn_more_about_openshift[Learn more about {product-title}].
```

Comment:
@opayne1 @obrown1205 WDYT? If keeping "the contents of", then at least suggest:

```
* Select the task that interests you from the contents of xref:../welcome/learn_more_about_openshift.adoc#learn_more_about_openshift[Learn more about {product-title}].
```
^ i.e., drop "the" and add period.

Reply:
That works for me! Thanks @adellape!